### ISSUE TITLE
Incompatible with Python 3.10

### ISSUE THREAD

**Issue by mcepl** [man-group/pytest-plugins#197]
Python 3.10 finally really eliminated all those classes in `collections` package which were long time ago moved to `collections.abc`. So, `from collections import Iterable` truly ceased to work.

This patch:
```diff
---
 pytest_verbose_parametrize.py |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

--- a/pytest_verbose_parametrize.py
+++ b/pytest_verbose_parametrize.py
@@ -1,4 +1,7 @@
-from collections import Iterable
+try:
+    from collections.abc import Iterable
+except (ImportError, ModuleNotFoundError):
+    from collections import Iterable
 from six import string_types, text_type
```
fixes the problem.

**Comment by eeaston** [comment]
Thanks for raising! We'll get this fixed.

**Comment by dasm** [comment]
It's actually fixed by now. It hasn't been released yet.

**Comment by anubhav96gupta** [comment]
Is there any time frame for this release?

**Comment by DanielNoord** [comment]
@eeaston Is there any chance of a release? Or do you need help fixing/reviewing anything that is blocking a release?

**Comment by DanielNoord** [comment]
@eeaston Gentle ping on this issue!

**Comment by jonbannister** [comment]
This has been released with 1.8.0, thanks for your patience.
